src/main/java/com/google/sps/servlets/DestinationDataServlet.java

```diff
@@ -44,10 +46,11 @@
   private static final String HINT3_PARAMETER = "hint3";
   private static final String OBSCURITY_PARAMETER = "obscurity";
   private static final String TAG_PARAMETER = "tag";
-  private static final String REDIRECT_URL = "/destination-data";
+  private static final String REDIRECT_URL = "/index.html";
 
-  // Temporarily stores the destination created by the user
-  public List<Destination> destinations = new ArrayList<>();
+  private static final DatastoreService DATASTORE = DatastoreServiceFactory.getDatastoreService();
```

Comment:
This isn't quite a constant since it's a service.  Don't use all caps in this case.

Reply:
I was trying to find a good explanation of why it shouldn't be in all caps but I was blanking out, so thanks for finding the passage. 👍